docs/proposals/pmm/01-vm-artifacts-redesign/01-vm-artifacts-redesign.md

```diff
@@ -0,0 +1,358 @@
+# VM artifacts for PMM
+
+## Summary
+
+Containers are a lightweight solution to run on any platform. One such platform is Virtual Machine. There is no additional benefit to having a custom Virtual Machine image for PMM.
+
+Modern Operation Systems adopted a new pattern of minimal VMs that are designed to run containers and are Cloud Native:
+
+- Fedora CoreOS (FCOS)
```

Comment:
@EvgeniyPatlan @talhabinrizwan addressing your feedback, we could recommend any OS of our choice
For example, Oracle 9 images do have cloud-init as well:

- https://docs.oracle.com/en-us/iaas/Content/Compute/References/images.htm#User
- https://docs.oracle.com/en-us/iaas/Content/Compute/References/images.htm#Oracle__linux-cloud-init

What those full-blown OS images don't provide are the following:
- minimalistic OS for running containers
- transactions
- auto-update

Actually, any modern VM OS would have one or another VM init functionality. So there is a plenty of choices.